### ISSUE TITLE
Camera upload automatically includes all new folders

### ISSUE THREAD

**Issue by Flat** [FilenCloudDienste/filen-mobile#156]
Currently if Camera upload is enabled it will automatically include any new folders created and start uploading images. I believe this setting should default to disabled for any new folders and a user should explicitly enable new folders to be included in the automatic upload.

**Comment by surfaceflinger** [comment]
+, photos sync should be paused by default like syncing on desktop client

**Comment by telrod11** [comment]
Also, allow deselection of "Parent" folder. Currently, Filen is trying to upload automatically, all the embedded art from my music library.   I now have to go in to 200+ individual folders to deselect this activity.

**Comment by youdontneedtoknow22** [comment]
so you expect me to go change for every new folder? no thanks

**Comment by Flat** [comment]
> so you expect me to go change for every new folder? no thanks

I could say the same to you.

**Comment by youdontneedtoknow22** [comment]
> > so you expect me to go change for every new folder? no thanks
> 
> I could say the same to you.

The point of synchronising photos is to not worry about them and back up every two weeks.
If I'll have that burden anyways, I would rather to back them offline

**Comment by Flat** [comment]
Yes, the point is to synchronize photos, not all the memes and other crap that is downloaded to my phone from various apps that I do not want backed up, that is the issue. By all means all photo folders from the camera should be backed up and auto added yes, but folders at root storage or other locations should not be.
